[5.9.0] - Add further information to the documentation. (#2904)
wso2/docs-is: en/docs/learn/configuring-sms-otp.md

            <td>redirectToMultiOptionPageOnFailure</td>
            <td>During a failed attempt enable redirect to the Multi Option Page where the user 
            can select the authentication mechanism.</td>
		</tr>
		<tr>
            <td>TokenExpiryTime</td>

>             <td>SMS OTP does not have a default validity period. Therefore, you must explicitly configure it by adding the <code>TokenExpiryTime</code> parameter. The value provided for the parameter is considered as seconds.</td>

Reviewer: ```suggestion
            <td>SMS OTP does not have a default validity period. Therefore, you must explicitly configure it by adding the <code>TokenExpiryTime</code> parameter. The value provided for the parameter is considered in seconds.</td>
```

>             <td>SMS OTP does not have a default validity period hence you will have to explicitly configure it by adding the <code>TokenExpiryTime</code> parameter. The value provided for the parameter is considered as seconds.</td>

Reviewer: ```suggestion
            <td>SMS OTP does not have a default validity period. Therefore, you must explicitly configure it by adding the <code>TokenExpiryTime</code> parameter. The value provided for the parameter is considered as seconds.</td>
```
